### ISSUE TITLE
App crash on map view

### ISSUE THREAD

**Issue by nandunbandara** [sliit-foss/covid19sl-mobile#8]
When the user navigates to the map view, the application crashes. 

Call stack:

`*** First throw call stack:
(
        0   CoreFoundation                      0x00007fff23c7127e __exceptionPreprocess + 350
        1   libobjc.A.dylib                     0x00007fff513fbb20 objc_exception_throw + 48
        2   CoreFoundation                      0x00007fff23c710bc +[NSException raise:format:] + 188
        3   Runner                              0x00000001063bf103 +[GMSServices checkServicePreconditions] + 182
        4   Runner                              0x00000001063bbf42 +[GMSServices sharedServicesSync] + 110
        5   Runner                              0x00000001063ab999 -[GMSMapView sharedInitWithServices:camera:forPlaces:] + 156
        6   Runner                              0x00000001063aae01 -[GMSMapView initWithFrame:camera:] + 123
        7   Runner                              0x00000001063aac25 +[GMSMapView mapWithFrame:camera:] + 104`

**Comment by yasaswick** [comment]
found it , might have caused if you have hot reloaded , I think you might have to run
"flutter clean" and try building it again.
And uninstall the previous app if you are on emulator

**Comment by nandunbandara** [comment]
Cool @yasaswick let me check.

**Comment by nandunbandara** [comment]
@yasaswick I was checking this on the iPhone simulator. Used a fresh build and still crashes. Let me check with the android emulator.

**Update**
@yasaswick the app is not crashing on android. 
![Screen Shot 2020-03-23 at 10 12 59 AM](https://user-images.githubusercontent.com/14901805/77282055-e64c5580-6cee-11ea-97f0-6e3f0381efc4.png)

**Comment by yasaswick** [comment]
Yeah sure , and for CI fails , I think most of them are there because we have some unused imports and for the -> info • Set literals weren't supported until version 2.2, but this code is required to be able to run on earlier versions • lib/screens/map.dart:11:24 • sdk_version_set_literal

we might have to update sdk version number in yaml or just use the syntax .. I will fix this in next request ..

**Comment by yasaswick** [comment]
> @yasaswick I was checking this on the iPhone simulator. Used a fresh build and still crashes. Let me check with the android emulator.
> 
> **Update**
> @yasaswick the app is not crashing on android.
> ![Screen Shot 2020-03-23 at 10 12 59 AM](https://user-images.githubusercontent.com/14901805/77282055-e64c5580-6cee-11ea-97f0-6e3f0381efc4.png)

I think the app crashes on iOS because I haven't added the Google Maps API key to the (pods ? I really don't know what they are called .. haha) , I only added Google Maps API key to Android manifest.
